src/plugins/intel_cpu/tests/functional/single_layer_tests/detection_output.cpp

```diff
@@ -8,16 +8,39 @@
 #include <common_test_utils/ov_tensor_utils.hpp>
 #include "test_utils/cpu_test_utils.hpp"
 #include "shared_test_classes/base/ov_subgraph.hpp"
+#include <dnnl.hpp>
+#include <cpu/x64/cpu_isa_traits.hpp>
 
 using namespace InferenceEngine;
 using namespace CPUTestUtils;
 using namespace ov;
 using namespace test;
+using namespace dnnl;
 
 namespace CPULayerTestsDefinitions {
 
 using namespace ov::test;
 
+unsigned get_cache_size(int level, bool per_core) {
```

Comment:
Is it possible to reuse [get_cache_size](https://github.com/openvinotoolkit/openvino/blob/b00fbd04c5c259672f65a29eeaec4b7ad6a0d433/src/plugins/intel_cpu/src/onednn/dnnl.h#L15) here to avoid code duplication?

Reply:
1. This function([get_cache_size](https://github.com/openvinotoolkit/openvino/blob/b00fbd04c5c259672f65a29eeaec4b7ad6a0d433/src/plugins/intel_cpu/src/onednn/dnnl.h#L15)) is just kind of wrap of ondnn call, and not exported as a cpu_plugin API. I am not sure it's make sense to export it as a plugin API just for testing.
2. I used to try export it and have some dependency issue.
So if it's need, I would like do it in a separate PR. what do you think?

Reply:
I'd suggest to add this static method to [CPUTestsBase](https://github.com/openvinotoolkit/openvino/blob/54609e7b72d9faa5a38fe7ac15c84aae77fbbc82/src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.hpp#L123) class to have opportunity to reuse it in other node tests at least. Moreover, this class already has the wrappers for `fmt2str` and `str2fmt` that are wraps of onednn call as well.
What do you think? If you would like to do in the separate PR and CI is green, let's do so. Otherwise I'd suggest to apply it in the current PR 😃

Reply:
Agree, have done in [6e572c1](https://github.com/openvinotoolkit/openvino/pull/19773/commits/6e572c1a880e4b58fcb11a4a53e0a8cd907b1302)